mains/create_davis_dataset.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import os
+from data.interp.davis.davis_data import DavisDataSet
+
+
+def main():
+    """
+    This program requires the following data structure at DAVIS/JPEGImages/(480p | 1080p):
+        <directory to shots 1>
+            <video_shot_1>
+            <video_shot_2>
+            ...
+            <video_shot_n>
+        <directory to shots 2>
+            <video_shot_1>
+            <video_shot_2>
+            ...
+            <video_shot_n>
+    """
+    parser = argparse.ArgumentParser()
+    add_args(parser)
+    args = parser.parse_args()
+
+    input_directory = os.path.join(args.directory, 'JPEGImages', '480p')
+    tf_records_directory = os.path.join(args.out_directory)
+    if not os.path.exists(tf_records_directory):
+        os.mkdir(tf_records_directory)
+
+    dataset = DavisDataSet(tf_records_directory, [[1]])
+    dataset.set_verbose(True)
+    dataset.preprocess_raw(input_directory, args.shard_size, validation_size=args.num_validation)
+
+
+def add_args(parser):
+    parser.add_argument('-d', '--directory', type=str,
+                        help='Directory of the unzipped DAVIS dataset.')
+    parser.add_argument('-o', '--out_directory', type=str,
```

Comment:
I think it's cleaner to specify an empty / non-existent folder that will be used just to keep the TFRecords. If you want to mix everything you should be able to, but I don't like that as a default.

Reply:
That's fine but it is now inconsistent with the flow dataset's commands. I am bothered more by inconsistency (even if its temporary) than defaulting to saving in the same directory.

Reply:
I do not want to accidentally spill TFRecords everywhere. If you are bothered by the inconsistency, it might be better for you to just change the flow dataset's command?